pkg/reconciler/revision/resources/deploy_test.go

```diff
@@ -103,6 +103,9 @@ var (
 		}, {
 			Name:  "QUEUE_SERVING_PORT",
 			Value: "8012",
+		}, {
+			Name:  "QUEUE_SERVING_TLS_PORT",
+			Value: "8112",
```

Comment:
We can just use `networking.BackendHTTPSPort`?

Reply:
`QUEUE_SERVING_PORT` does not the const so I leave it as it is.